src/main/java/com/cgdpd/library/controller/BookController.java

```diff
@@ -35,4 +41,13 @@ public CreateBookResponseDTO createBook(@RequestBody CreateBookRequestDTO reques
     public DetailedBookDTO getBook(@PathVariable Isbn13 isbn13) {
         return bookService.getDetailedBookByIsbn13(isbn13);
     }
+
+    @GetMapping
+    @ResponseStatus(HttpStatus.OK)
+    public PagedResponse<DetailedBookDTO> searchBook(
+          @RequestParam(required = false) Optional<SearchBookCriteria> searchBookCriteria,
```

Comment:
Remove the annotation, and play around by removing one parameter or the other, problem has something to do with PaginationCriteria receiving invalid values